runtime/shared_common/CacheMap.cpp

```diff
@@ -984,6 +984,8 @@ SH_CacheMap::updateROMSegmentListForCache(J9VMThread* currentThread, SH_Composit
 			currentROMClass += currentROMSize;
 		}
 		currentSegment->heapAlloc = cacheAlloc;
+		VM_AtomicSupport::writeBarrier();
+		Trc_SHR_CM_updateROMSegmentList_NewHeapAlloc(currentThread, currentSegment, cacheAlloc);
```

Comment:
Do we need this enabled by default (level 2)?

Reply:
Changed to level 4